server/src/test/java/io/crate/planner/UpdatePlannerTest.java

```diff
@@ -176,6 +176,7 @@ public void testMultiValueSubQueryWithinSingleValueSubQueryDoesNotInheritSoftLim
             "update users set ints = (" +
                 "   select count(id) from users where id in (select unnest([1, 2, 3, 4])))");
         assertThat(plan.rootPlan, instanceOf(UpdatePlanner.Update.class));
+        assertThat(plan.type(), is(Plan.StatementType.UPDATE));
```

Comment:
Same here, looks like `test_classify_multiphase_update_statement` already covers the issue.